templates/base/packages/nextjs/tsconfig.json.template.mjs

```diff
@@ -20,9 +22,23 @@
     "plugins": [
       {
         "name": "next"
-      }
-    ]
+      },
+      ...(extraPlugins?.[0] || [])
+    ],
+    ...(extraCompilerOptions?.[0] || {})
```

Comment:
same, `extraCompilerOptions` array always exists since default value exists. We can make default value `{}` (but it will work even with `null`) and use `...extraCompilerOptions[0]` here